src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -40,5 +40,140 @@ $duo-tone-lighten: #000000;
 }
 
 .facility-satisfaction-tile {
-  width: 147px;
+  width: 50%;
+
+  @include media($medium-large-screen) {
+    width: 147px;
+  }
+}
+
+.basic-facilities-list {
+  .facility {
+    &:last-child {
+      margin-bottom: 24px;
+    }
+  }
+}
+
+.region-img,
+.news-img {
+  width: 100%;
+  height: 165px;
+  object-fit: cover;
+
+  @include media($small-screen) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.facility-img {
+  width: 100%;
+  height: 164px;
+  object-fit: cover;
+
+  @include media($small-screen) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.news-img {
+  @include media($small-screen) {
+    height: 180px;
+    width: 100%;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+  }
+}
+
+.story-detail-img {
+  width: 100%;
+  height: 164px;
+  object-fit: cover;
+
+  @include media($medium-large-screen) {
+    width: 100%;
+    height: 295px;
+    object-fit: cover;
+  }
+}
+
+.event-detail-img {
+  width: 100%;
+  height: 159px;
+  object-fit: cover;
+
+  @include media($medium-large-screen) {
+    width: 100%;
+    height: 195px;
+    object-fit: cover;
+  }
+}
+
+.region-list {
+  .region-grid {
+    width: 100%;
+    padding-right: initial;
+    padding-left: initial;
+    &:not(:only-child):last-child {
+      padding-right: initial;
+      padding-left: initial;
+    }
+
+    &.stories-list {
+      width: 100%;
+    }
+
+    @include media($small-screen) {
+      width: 50%;
+      padding-right: 12px;
+      padding-left: 0;
+      &:not(:only-child):last-child{
+        padding-right: 0;
+        padding-left: 12px;
+      }
+
+      &.stories-list {
+        width: 100%;
+      }
+    }
+
+    @include media($medium-large-screen) {
+      width: 65%;
+      &.event,
+      &.facility {
+        width: 50%;
+      }
+
+      &:not(:only-child):last-child {
+        &.event,
+        &.facility {
+          width: 50%;
+        }
+        width: 32%;
+      }
+    }
+  }
+}
+
+
+// to push items at the end of an element wrapped in display: flex to the bottom
+.vads-u-margin-top--auto {
```

Comment:
This makes sense to me as a helper, but I don't think we should create new classes with the `vads` prefix in vets-website code. That should be limited to the design system. Can we change the name?

cc: @jaredcunha-usds

Reply:
ok